meson_options.txt

```
@@ -0,0 +1,11 @@
+option(
+  'utils',
+  type : 'boolean',
+  value : false,
+  description : 'Enable building utils')
+
+option(
+  'jpeg',
+  type : 'boolean',
+  value : true,
+  description : 'Enable building with jpeg')
```

```suggestion
option(
  'jpeg',
  type : 'feature',
  value : 'enabled',
  description : 'Enable building with jpeg')
```

See the documentation for feature options: https://mesonbuild.com/Build-options.html#features